FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.Core/Jobs/Models/AzureStorage/OrchestratorJobStatusEntity.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure;
+using Azure.Data.Tables;
+
+namespace Microsoft.Health.Fhir.Synapse.Core.Jobs.Models.AzureStorage
+{
+    public class OrchestratorJobStatusEntity : ITableEntity
+    {
+        public string PartitionKey { get; set; }
+
+        public string RowKey { get; set; }
+
+        public DateTimeOffset? Timestamp { get; set; }
+
+        public ETag ETag { get; set; }
+
+        /// <summary>
+        /// Statistic result of orchestrator job.
+        /// </summary>
+        public string StatisticResult { get; set; }
+
+        /// <summary>
+        /// The corresponding orchestrator job id of current trigger
+        /// </summary>
+        public long GroupId { get; set; } = 0;
```

Comment:
seems `GroupId` is useless?

Reply:
It will be clearer and more convenient to use the entity in table? (e.g., observe and query)